docs/concepts/workloads/vm.md

```diff
@@ -360,9 +561,75 @@ Deployed VMs inherit the storage defined in the `VirtualMachineImage`. To provid
 
 6. Mount the disk and begin using it.
 
-## Power States
+#### Volume Status
+
+The field `status.volumes` described the observed state of a `VirtualMachine` resource's volumes, including information about the volume's usage and encryption properties:
+
+=== "Volume Properties"
+
+    | Name | Description |
+    |------|-------------|
+    | `attached` | Whether or not the volume has been successfully attached to the `VirtualMachine`. |
+    | `crypto` | An optional field set only if the volume is encrypted. |
+    | `diskUUID` | The unique identifier of the volume's underlying disk. |
+    | `limit` | The maximum amount of space that may be used by this volume. |
+    | `name` | The name of the volume. For managed disks this is the name from `spec.volumes` and for classic disks this is the name of the underlying disk. |
+    | `type` | The total storage space occupied by this VirtualMachine that is not shared with any other `VirtualMachine`. |
```

Comment:
This seems wrong and should be https://github.com/vmware-tanzu/vm-operator/blob/main/api/v1alpha3/virtualmachine_storage_types.go#L111?

Also, should we include the [Error](https://github.com/vmware-tanzu/vm-operator/blob/main/api/v1alpha3/virtualmachine_storage_types.go#L145-L147) field here as well?